packages/compass-global-writes/src/store/reducer.ts

```diff
@@ -561,6 +598,23 @@ export const fetchClusterShardingData =
     }
   };
 
+export const resumeManagedNamespace = (): ReturnType<typeof createShardKey> => {
+  return async (dispatch, getState) => {
+    const { shardKey } = getState();
+    if (!shardKey) {
+      throw new Error('Cannot resume managed namespace without a shardKey');
+    }
+    const data: CreateShardKeyData = {
+      customShardKey: shardKey.fields[1].name,
+      isShardKeyUnique: shardKey.isUnique,
+      isCustomShardKeyHashed: shardKey.fields[1].type === 'HASHED',
+      numInitialChunks: null, // default
+      presplitHashedZones: false, // default
+    };
+    await dispatch(createShardKey(data));
```

Comment:
When I resume sharding, it eventually transitions to `SHARDING` status and if I cancel that process, it takes me back to `UNSHARDED` status but it should transition to `INCOMPLETE_SHARDING_SETUP` status, right?

Reply:
you're right, I missed that one! thanks a lot!